README.fr-FR.md

```diff
@@ -0,0 +1,16 @@
+<img width="150" height="150" align="left" style="float: left; margin: 0 10px 0 0;" alt="MS-DOS logo" src="https://github.com/Microsoft/MS-DOS/blob/master/msdos-logo.png">   
+
+# Code source de MS-DOS v1.25 et v2.0
+Ce dépôt contient le code source original et les binaires compilés de MS-DOS v1.25 et MS-DOS v2.0.
+
+Ce sont les mêmes fichier que [ceux partagés initialement au Computer History Museum le 25 Mars 2014]( http://www.computerhistory.org/atchm/microsoft-ms-dos-early-source-code/) et sont (re)publiés sur ce dépôt pour les rendre plus accessibles et plus simple à référencer dans des travaux ou articles extérieurs, et pour permettre l'exploration et l'expérimentation pour les personnes intéressées par les première générations de systèmes d'exploitation pour PC.  
+
+# Licence
+Tous les fichiers de ce dépôt sont mis à disposition selon la [Licence MIT (OSI)]( https://fr.wikipedia.org/wiki/Licence_MIT) en accord avec le [fichier LICENCE](https://github.com/Microsoft/MS-DOS/blob/master/LICENSE.md) stocké à la racine de ce dépôt.
+
+# Contribuez!
+Les fichiers sources de ce dépôt sont mis à disposition pour référence historique et ne seront pas modifiés, donc s'il vous plait **n'envoyer pas** de Pull Requests suggérant des modifications aux fichiers sources, mais forkez ce dépôt et expérimentez 😊.  
+
+Si, toutefois, vous souhaitiez soumettre du contenu additionel ou des modifications à des fichiers autres que les sources, (ex., this README), soumettez via une PR, nous ferons une revue et la prendrons en compte.
```

Comment:
Not translated parenthesis : 
`(par exemple ce README)`

Reply:
Thanks for the review !